tools/clang/test/HLSLFileCheck/d3dreflect/bitfield_reflection.hlsl

```diff
@@ -0,0 +1,77 @@
+// RUN: %dxc -E main -T ps_6_0 -HV 2021 -Vd -validator-version 0.0 %s | %D3DReflect %s | FileCheck %s
+
+// Make sure bitfiled info is saved.
+// TODO - make better dump. Now, Rows = 0 means bitfield. Columns is bit width. Elements is bit offset.
```

Comment:
Comment out of date

Reply:
To clarify:
- `Rows = 0 means bitfield` shouldn't be the indicator, since it uses `D3D_SVC_BIT_FIELD` now.  However, I still see that Rows are 0 for bitfields.  Should they be?  I'm not sure it matters, but it's not necessary anymore.
- `Elements is bit offset` - this is `Offset` now.

Reply:
Xiang, are you going to address this comment?

Reply:
Fixed. The outdated comment is removed. Rows will be 1 now.